parent/pom.xml

```diff
@@ -47,7 +47,7 @@
         <wsdl4j.version>1.6.3</wsdl4j.version>
         <xalan.version>2.7.2</xalan.version>
         <xz.version>1.9</xz.version>
-        <xmlsec.version>2.3.0</xmlsec.version>
+        <xmlsec.version>2.3.1-SNAPSHOT</xmlsec.version>
```

Comment:
Change to 3.0.0-SNAPSHOT after applying https://github.com/apache/santuario-xml-security-java/pull/63

I see some errors:

java.lang.ExceptionInInitializerError
	at org.apache.wss4j.stax.test.VulnerabliltyVectorsDecompressedBytesTest.setup(VulnerabliltyVectorsDecompressedBytesTest.java:44)
Caused by: java.lang.RuntimeException: Implementation of Jakarta XML Binding-API has not been found on module path or classpath.

Reply:
Requested change made.
Test failures resolved.